src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+    public CEO(String name)
+    {
+        this.name=name;
+    }
+    private String name;
+    private ArrayList<AbstractEmployee> employees_list = new ArrayList();
```

Comment:
It would be better to define this list like that:

```
private List<AbstractEmployee> employees_list = new ArrayList();
```

Because from user's point of view it's not really important if it's an `ArrayList`, `LinkedList`, `CopyOnWriteArrayList` or some other implementation. Now you're bound to a concrete implementation which is not very flexible.